sdk/metric/metricdata/data.go

```diff
@@ -240,3 +240,54 @@ type Exemplar[N int64 | float64] struct {
 	// be empty.
 	TraceID []byte `json:",omitempty"`
 }
+
+// Summary metric data are used to convey quantile summaries,
+// a Prometheus (see: https://prometheus.io/docs/concepts/metric_types/#summary)
+// data type.
+//
+// These data points cannot always be merged in a meaningful way. The Summary
+// type is only used by bridges from other metrics libraries, and cannot be
+// produced using OpenTelemetry instrumentation.
+type Summary struct {
+	// DataPoints are the individual aggregated measurements with unique
+	// attributes.
+	DataPoints []SummaryDataPoint
+}
+
+func (Summary) privateAggregation() {}
+
+// SummaryDataPoint is a single data point in a timeseries that describes the
+// time-varying values of a Summary metric.
+type SummaryDataPoint struct {
+	// Attributes is the set of key value pairs that uniquely identify the
+	// timeseries.
+	Attributes attribute.Set
+
+	// StartTime is when the timeseries was started.
+	StartTime time.Time
+	// Time is the time when the timeseries was recorded.
+	Time time.Time
+
+	// Count is the number of updates this summary has been calculated with.
+	Count uint64
+
+	// Sum is the sum of the values recorded.
+	Sum *float64
+
+	// (Optional) list of values at different quantiles of the distribution calculated
+	// from the current snapshot. The quantiles must be strictly increasing.
+	QuantileValues []ValueAtQuantile
+}
+
+// ValueAtQuantile the value at a given quantile of a distribution.
+type ValueAtQuantile struct {
+	// The quantile of a distribution.
+	//
+	// Must be in the interval [0.0, 1.0].
+	Quantile float64
```

Comment:
nit: what do you think about calling this `QuantileValue`?

```suggestion
// QuantileValue the value at a given quantile of a distribution.
type QuantileValue struct {
	// Quantile is the quantile of this value.
	//
	// Must be in the interval [0.0, 1.0].
	Quantile float64
```

Reply:
updated